pkg/controllers/node/node_syncer.go

```diff
@@ -74,14 +74,18 @@ func (c *NodeController) OnUpdates(updates []bapi.Update) {
 					// It has a node reference - get that Kubernetes node, and if
 					// it exists perform a sync.
 					obj, ok, err := c.indexer.GetByKey(kn)
+
+					// Just log, we want to execution for this event to
```

Comment:
I think this comment is in the wrong place - it should be within one of the failure case branches, rather than at the top-level, since the comment doesn't apply if there is no failure.

Reply:
I left the comment there but made it more generally about the whole if/else